src/languages/en.ts

```diff
@@ -1637,6 +1637,10 @@ export default {
         },
     },
     privatePersonalDetails: {
+        enterLegalName: 'Enter legal name',
```

Comment:
```suggestion
        enterLegalName: 'What's your legal name?',
```

Reply:
@jamesdeanexpensify wanted to update these titles to questions, so I've suggested the changes for the English ones @shubham1206agra.